spec.bs

```diff
@@ -3097,6 +3194,20 @@ dictionary ReportWinBrowserSignals : ReportingBrowserSignals {
     bidding signals server
 </dl>
 
+<xmp class="idl">
+dictionary DirectFromSellerSignalsForBuyer {
+  any auctionSignals = null;
+  any perBuyerSignals = null;

```

Comment:
These will all be null if they are not defined anyways, since they are not [required members](https://webidl.spec.whatwg.org/#required-dictionary-member). So I think we can remove the ` = null` from these.

Also, can we pick a more specific type? These are super super general and without context it is hard to understand what they actually represent. What are they in practice, and why are they `any` here?

Reply:
OK that's reasonable then. Also I apologize, but I have to walk back my previous statement. If these values are omitted, they will be `undefined` by default, not `null`. So if you really want them to be `null` by default (i.e.., when these members are not explicitly provided) and _never_ `undefined` then you will need to put ` = null;` back... up to you, I'm not sure which you need or how the Chromium implementation handles that, but I just wanted to call that out.

Reply:
Oops, these need to be null. Since we've already merged, I filed https://github.com/WICG/turtledove/pull/861 to fix.

Reply:
Thanks for fixing it. Sorry I was out last Friday, so didn't saw this comment.